msgpack/ext.py

```diff
@@ -5,10 +5,7 @@
 import struct
 
 
-try:
-    _utc = datetime.timezone.utc
-except AttributeError:
-    _utc = datetime.timezone(datetime.timedelta(0))
+_utc = datetime.timezone.utc
```

Comment:
Maybe using it directly at all places where it is needed would be prettier.

Reply:
I inlined the `_utc` in function and tests, under `utc` name.